pkg/kubelet/pleg/generic.go

```diff
@@ -275,6 +288,15 @@ func (g *GenericPLEG) relist() {
 				metrics.PLEGDiscardEvents.Inc()
 				klog.ErrorS(nil, "Event channel is full, discard this relist() cycle event")
 			}
+			// Log exit code of containers when they finished in a particular event
+			if events[i].Type == ContainerDied {
+				containerID, ok := events[i].Data.(string)
+				if ok {
+					if exitCode, ok := containerExitCode[containerID]; ok {
+						klog.V(2).Infof("GenericPLEG: container finished: container ID: %v, exit code: %v", containerID, exitCode)
```

Comment:
would the pod ID be useful here as well?

Reply:
yeah, it might be useful for filtering pleg logs by Pod ID if we are only interested to a particular pod. Will add pod ID as well